lib/charms/prometheus_k8s/v0/prometheus_remote_write.py

```diff
@@ -569,26 +569,46 @@ def __init__(self, *args):
     If the syntax of a rule is invalid, the `MetricsEndpointProvider` logs an error and
     does not load the particular rule.
 
-    To avoid false positives and negatives in the evaluation of your alert rules,
-    you must always add the `%%juju_topology%%` token as label filters in the
-    PromQL expression, e.g.:
-
-        alert: UnitUnavailable
-        expr: up{%%juju_topology%%} < 1
-        for: 0m
-        labels:
-            severity: critical
-        annotations:
-            summary: Unit {{ $labels.juju_model }}/{{ $labels.juju_unit }} unavailable
-            description: >
-            The unit {{ $labels.juju_model }} {{ $labels.juju_unit }} is unavailable
-
-    The `%%juju_topology%%` token will be replaced with label filters ensuring that
+    To avoid false positives and false negatives the library will inject label filters
+    automatically in the PromQL expression. For example if the charm provides an
+    alert rule like this one:
```

Comment:
Should we narrow the example to the `expr` only?
```suggestion
    alert rule with an `expr` like this one:
```